### ISSUE TITLE
Doesn't Start

### ISSUE THREAD

**Issue by guiselig** [ModernFlyouts-Community/ModernFlyouts#1317]
### ModernFlyouts Version

0.9.3.0

### Windows version

Windows 11 Home Single Language 23H2

### OS build

22631.3880

### Platform

x64

### Area(s) with issue?

General

### Steps to reproduce

I can't open the app. I downloaded from the github page and when I tried to open it, nothing happened. I also executed it as a admin, but nothing happened as well. Then I uninstalled the app and reinstalled it from the microsoft store, but the same things keep happening

### ❌ Actual Behavior

Written above

**Comment by prakharb5** [comment]
Can you try the beta version here on github?

**Comment by guiselig** [comment]
Downloaded the beta 4 and it worked.

**Comment by prakharb5** [comment]
Great

**Comment by yelfaramawy** [comment]
> Can you try the beta version here on github?
@prakharb5 
I tried the v0.10.0-beta.4 version, but unfortunately getting the app won't start.

**Comment by prakharb5** [comment]
What happens when you start the app from start menu?

**Comment by yelfaramawy** [comment]
It shows the app icon on the system tray icons, then it disappears again after like 3 secs.

I noticed that there are some apps related to the app in the start menu like (Settings and Restore legacy flyout) but if I click on any of them it says the item is unavailable it might be deleted or moved. I tried to reinstall many times and also installed the beta version like you said but it didn't help.
